### PR TITLE
Browser SDK Loader Shim Support Update

### PR DESCRIPTION
**Pull request by JacksonWeber**
* Update Shim Support table to include the browser SDK loader for the distro.
* Remove contradiction regarding Azure Functions in the Distro.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,10 +173,10 @@ And invoked via `appInsights.<METHOD_NAME>`
 | ----------------------------|------------------------------------------------------------------------------------------------------------|
 | setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
 | setAutoCollectHeartbeat     | Heartbeat is not supported in either the distro or the shim.|
-| enableWebInstrumenatation   | WebInstrumentation is not supported in either the distro or the shim. |
+| enableWebInstrumenatation   | WebInstrumentation is not supported in the shim, but is available in the distro as `enableBrowserSdkLoader` |
 | setAutoDependencyCorrelation| Turning off autoDependencyCorrelation is not supported by either the distro or the shim. |
 | setUseDiskRetryCaching      | While enabling/disabling offline storage is supported, setting the resend interval or the maxBytesOnDisk values are not supported in the shim or the distro. |
-| setAutoCollectIncomingRequestAzureFunctions | Auto collection of Azure Functions is not supported by the shim or the distro. |
+| setAutoCollectIncomingRequestAzureFunctions | Not supported by the shim. Migrate to the distro to use auto collection of Azure Functions. |
```
Comment (hectorhdzg) [inline README.md:179]: Auto collection of incoming requests is not supported in distro either

File: README.md [hunk 2]
```diff
@@ -173,10 +173,10 @@ And invoked via `appInsights.<METHOD_NAME>`
 | ----------------------------|------------------------------------------------------------------------------------------------------------|
 | setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
 | setAutoCollectHeartbeat     | Heartbeat is not supported in either the distro or the shim.|
-| enableWebInstrumenatation   | WebInstrumentation is not supported in either the distro or the shim. |
+| enableWebInstrumenatation   | WebInstrumentation is not supported in the shim, but is available in the distro as `enableBrowserSdkLoader` |
```
Comment (hectorhdzg) [inline README.md:176]: Distro is not used in supported configuration and not explained anywhere in this readme, better to use  "Azure Monitor OpenTelemetry", you can also support old config in shim and change it to correct new value like other configs here

Comment (JacksonWeber) [reply README.md:176]: I'll update that language. And I'll create a task for updating the shim to use the browser SDK loader in the distro once it's released.

File: README.md (hunk 1 above)
Comment (JacksonWeber) [reply README.md]: Corrected and addressed the contradiction in the JSON config table.

**Review by hectorhdzg — APPROVED** [review]
LGTM

**Pull request merged**